experimental-examples/kitchen-sink/content/author/Napolean.md

```diff
@@ -1,4 +1,5 @@
 ---
-name: Napolean
+name: melbourne - Napolean
```

Comment:
**issue**: If this test data is to be retained, then please capitalize the placenames "Melbourne" and "Sydney".

Reply:
I guess we should also fix the spelling of "Napolean" to _Napoleon_ as well 🙏

Reply:
Done - fix spelling mistake